src/QueryNotFoundException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * @file
```

Comment:
Actually, I am not sure if we even need a dedicated exception type. What do you think?